content/documentation/install-uninstall.md

```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
 {{< /q-class >}}
 
 ## macOS Installation
 
-Open your Terminal {{< q-def "command-line shell" >}} (found in your Applications/Utilities folder or by pressing Command–Space Bar and type “Terminal” ) and follow the steps below to first install the support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+
+### Quick Install
+
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 3–5 in the Full Install guidelines below.
+
+1. In Terminal, install Apple’s Xcode with: `xcode-select --install`
+2. Download and install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+3. In Terminal, install Quire with: `npm install --global @thegetty/quire-cli`
+4. Confirm installation: `quire --help`
+
+The Quire installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation, as would seeing "command not found" after entering `quire --help` in step 4. Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```

Comment:
Can we have this match lines 66 and 68? I'm fine with it if this is your preference, I just like the consistent messaging and explanation of the commands.

Reply:
Match in what way? I saw 66 and 68 as a fuller version of the process. I used here verbatim on line 66, but I can't change 68 because they haven't yet run `quire --help`

Reply:
I think Step. 4 is a little confusing because we aren't telling them what to expect by running that command other than you might see "command not found." 

I can see how it might be too wordy but I feel like we should incorporate "You should see  a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire." I can imagine some people might get confused when they run that command and a bunch of text pops up in their terminal.